source/about_me.html.haml

```diff
@@ -1,20 +1,20 @@
   %br
-  %p My field of expertise is originally biology and I am very passionate about cultures of bacteria, yeast and virus, DNA sequencing, all the new and old molecular biology techniques and I still follow the latest updates within the scientific community.
+  %p My field of expertise is originally biology and I am very passionate about cultures of bacteria, yeast &amp; viruses, DNA sequencing; all the new &amp; old molecular biology techniques and I still follow the latest updates within the scientific community.
   %br
-  %p I've started my journey into programming last year with my first Java book and from that moment I've been anchored to this world driven by a really strong passion for coding and creating programs. After trying to make a living out of my passion for Life Science going through many difficulties, the bootcamp for me has been the opportunity I needed to give myself a second chance at working with something I love.
+  %p I started my journey into programming last year with my first Java book and from that moment I've been anchored to this world driven by a really strong passion for coding and creating programs. After having many difficulties trying to make a living out of my passion for Life Sciences, the bootcamp for me has been the opportunity I needed to give myself a second chance at working with something I love.
   %br
-  %p I am a very creative and curious person, who is not just cut for any kind of job. I need for my job to be stimulating, challenging and to give me the opportunity to learn something new every day. Computer Science was the only natural choice to be made for me after facing the cruel competition in the field of Life Science. The reason I chose to be part of a bootcamp instead of going through the academic path all over again resides in the fact that I wanted to do something that could really change my life. I knew it would have been hard and trivial, demanding and stressful but that with the right amount of patience and determination I could succeed and hopefully gain some practical experience along the way.
+  %p I am a very creative and curious person, who is not cut out for just any kind of job. I need my job to be stimulating, challenging and to give me the opportunity to learn something new every day. Computer Science was the only natural choice to be made for me after facing the cruel competition in the field of Life Science. The reason I chose to be part of a bootcamp instead of going through the academic path all over again resides in the fact that I wanted to do something that could really change my life. I knew it would have been hard and trivial, demanding and stressful but that with the right amount of patience and determination I could succeed and hopefully gain some practical experience along the way.
```

Comment:
Again the reference to "cruel competition" feels a little oblique - I've worked in academia so you know what I mean, but industry can be competitively cruel too.   Again something to think about whether you want to say more or less about what made you want to leave academia.

Also the following sentence:

> I knew it would have been hard and trivial, demanding and stressful

seems odd because trivial feels like the odd one out